acquia_lift_profiles/qunit/tests.js

```diff
@@ -139,7 +139,16 @@ QUnit.asyncTest( "init test", function( assert ) {
 });
 
 
+QUnit.test("Get context values with 'do not track' cookie", function( assert ) {
+  $.cookie('tc_dnt', 'true', {path:'/'});
+  expect(1);
+  var contextResult = Drupal.personalize.visitor_context.acquia_lift_profiles_context.getContext({'segment1':'segment1'});
+  assert.ok($.isEmptyObject(contextResult), 'Context result must be an empty object');
+  $.cookie('tc_dnt', null, {path:'/'});
+});
+
 QUnit.asyncTest("Get context values no cache", function( assert ) {
+  $.cookie('tc_dnt', null, {path:'/'});
```

Comment:
We should just add this to the tearDown function for the module, if there is a module for these tests. If not we should add one :)